src/Xrefcheck/CLI.hs

```diff
@@ -109,7 +109,7 @@ totalParser = asum
   ]
 
 versionOption :: Parser (a -> a)
-versionOption = infoOption ("xrefcheck-" <> showVersion version) $
+versionOption = infoOption ("xrefcheck-" <> (showVersion version)) $
```

Comment:
Some mess happened: you removed these brackets and then added them back.

Reply:
Wtf, git failed into move + fix.